src/renderer/dx/DxRenderer.cpp

```diff
@@ -2147,6 +2177,8 @@ void DxEngine::SetSelectionBackground(const COLORREF color) noexcept
                                         GetGValue(color) / 255.0f,
                                         GetBValue(color) / 255.0f,
                                         0.5f);
+
+    _selectionBackground = D2D1::ColorF(D2D1::ColorF::Red);

```

Comment:
yott

Reply:
Yeah, thanks. That was temporary so I could highlight things with a selection and see if they were off on the top or bottom or both very easily by pushing my face into the monitor.